libs/judicial-system/types/src/lib/eventLog.ts

```diff
@@ -9,6 +9,11 @@ export enum EventType {
   INDICTMENT_REVIEWED = 'INDICTMENT_REVIEWED',
   CASE_RECEIVED_BY_COURT = 'CASE_RECEIVED_BY_COURT',
   INDICTMENT_SENT_TO_PUBLIC_PROSECUTOR = 'INDICTMENT_SENT_TO_PUBLIC_PROSECUTOR',
+  INDICTMENT_SENT_TO_FMST = 'INDICTMENT_SENT_TO_FMST',
```

Comment:
Is this still in use since the creation of the defendant event log?

Reply:
Agree with Una.